Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,247 @@
+         * Gets and sets the highlight color multiplied with the feature's color.  When
+         * this is white, the feature's color is not changed. This is set for all features
+         * when a style's color is evaluated.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Color}
+         *
+         * @default {@link Color.WHITE}
+         */
+        color : {
+            get : function() {
+                if (!defined(this._color)) {
+                    this._color = new Color();
+                }
+                return this._content.batchTable.getColor(this._batchId, this._color);
+            },
+            set : function(value) {
+                this._content.batchTable.setColor(this._batchId, value);
+            }
+        },
+
+        /**
+         * Gets the content of the tile containing the feature.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Cesium3DTileContent}
+         *
+         * @readonly
+         * @private
+         */
+        content : {
+            get : function() {
+                return this._content;
+            }
+        }
+    });
+
+    /**
+     * Returns whether the feature contains this property. This includes properties from this feature's
+     * class and inherited classes when using a batch table hierarchy.
+     *
+     * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/BatchTable#batch-table-hierarchy}
+     *
+     * @param {String} name The case-sensitive name of the property.
+     * @returns {Boolean} Whether the feature contains this property.
+     */
+    Cesium3DTileFeature.prototype.hasProperty = function(name) {
+        return this._content.batchTable.hasProperty(this._batchId, name);
+    };
+
+    /**
+     * Returns an array of property names for the feature. This includes properties from this feature's
+     * class and inherited classes when using a batch table hierarchy.
+     *
+     * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/BatchTable#batch-table-hierarchy}
+     *
+     * @returns {String[]} The names of the feature's properties.
+     */
+    Cesium3DTileFeature.prototype.getPropertyNames = function() {
+        return this._content.batchTable.getPropertyNames(this._batchId);
+    };
+
+    /**
+     * Returns the value of the feature's property with the given name. This includes properties from this feature's
+     * class and inherited classes when using a batch table hierarchy.
+     * <p>
+     * The value is copied before being returned.
+     * </p>
+     *
+     * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/BatchTable#batch-table-hierarchy}
+     *
+     * @param {String} name The case-sensitive name of the property.
+     * @returns {*} The value of the property or <code>undefined</code> if the property does not exist.
+     *
+     * @example
```

Comment:
Same comment as above, my preference would be to remove all new `@example` doc references and instead link to targeted Sandcastle examples.  Can be part of a larger effort post merge, since our doc needs TLC as a whole.